tests/test_h264.py

```diff
@@ -128,7 +128,7 @@ def test_encoder_large(self):
         # delta frame
         frame = self.create_video_frame(width=1280, height=720, pts=3000)
         payloads, timestamp = encoder.encode(frame)
-        self.assertEqual(len(payloads), 1)
+        self.assertGreaterEqual(len(payloads), 1)
```

Comment:
Is this actually related to Python 3.13 support? If it's not, please move it to a separate PR.

Reply:
I'm yanking this change, but feel free to reintroduce it in a separate PR.